lib/ansible/cli/__init__.py

```diff
@@ -597,8 +597,7 @@ def parse(self):
 
             else:
                 # set default if it exists
-                if os.path.exists(C.DEFAULT_HOST_LIST):
-                    self.options.inventory = [C.DEFAULT_HOST_LIST]
+                self.options.inventory = [host_list for host_list in C.DEFAULT_HOST_LIST]
```

Comment:
since C.DEFAULT_HOST_LIST is already a list ... not sure we need any of this just:
```python
self.options.inventory = C.DEFAULT_HOST_LIST
```

Reply:
indeed.